content/next/index.md

```diff
@@ -155,17 +160,17 @@ The first draft of this specification has been written in collaboration with som
 * [php-commitizen](https://github.com/damianopetrungaro/php-commitizen): a tool built to create commit messages following the Conventional Commit specs. 
 Configurable and usable for PHP projects as a composer dependency or usable globally for non-PHP projects.
 * [conform](https://github.com/autonomy/conform): a tool that can be used to enforce policies on git repositories, including conventional commits.
+* [standard-version](https://github.com/conventional-changelog/standard-version): Automatic versioning and CHANGELOG management, using GitHub's new squash button and the recommended Conventional Commits workflow.
```

Comment:
Should this be in this PR?

Reply:
I didn't think it was the end of the world? It was bugging me that `standard-version` was in the wrong section of the footer, and thought I'd bump it up; can pull into another pull request if you prefer.

Reply:
It's fine :) Let's just create a different commit for that one and I am totally fine with it 😄